src/metabase/cmd/env_var_dox.clj

```diff
@@ -0,0 +1,110 @@
+(ns metabase.cmd.env-var-dox
+  "Code to generate docs for environment variables. You can generate
+  docs by running: `clojure -M:ee:run environment-variables-documentation`"
+  (:require [clojure.java.classpath :as classpath]
+            [clojure.java.io :as io]
+            [clojure.string :as str]
+            [clojure.tools.namespace.find :as ns.find]
+            [clojure.tools.reader.edn :as edn]
+            [metabase.models.setting :as setting]
+            [metabase.util :as u]))
+
+(def env-vars-not-to-mess-with
+  "Flamber advises that people avoid touching these environment variables."
+  (set (edn/read-string (slurp (io/file "src/metabase/cmd/resources/env-vars-to-avoid.edn")))))
```

Comment:
Instead of sticking them in a separate list, why not make this information part of the `:defsetting` macro itself? You can call it `:skip-doc` or something. Idk. But then the information is attached to the Setting definition instead of living in some mystery file somewhere

Reply:
Or maybe if we were going to have a `:doc` key or something that we use for other stuff like I suggested in the issue, maybe you can just say `:doc false`.

```clj
(defsetting my-setting
  :doc {:description "Flamber says always set this", :added "0.45.0"})

(defsetting do-not-document
  :doc false)
```

Maybe that works?

Reply:
100% for this instead of a separate list somewhere. And it seems quite smart if we could have separate doc info, so internal dev comment doesn't have to be the same as doc comment - and gives the added benefit of additional stuff like `:added` and whatever might come in the future, but there's also a few cases where the default value is coming from upstream (like some Jetty vars)